src/app/modules/core/components/map/map.component.ts

```diff
@@ -218,6 +220,11 @@ export class MapComponent extends deepCopy implements OnInit, AfterViewInit, OnC
     }
   }
 
+  public getLatLong(evnt: any) { 
+    this.evnt = evnt.latlng;
+    this._mousePosition = evnt.latlng;
```

Comment:
nit-picky, but do we need to save `this.evnt`? If not, you could remove line 224.

Reply:
It was weird. I was trying to use this.evnt but it was holding the entire evnt, not only the latlng. Perhaps I don't understand the way these things are assigned. I did remove line 224 AND I also removed this same thing from the onMouseClick function and the property declaration as they don't appear to be needed. Code is sparkling now. ;-)